core/eval.go

```diff
@@ -193,6 +197,23 @@ func evalSET(args []string) []byte {
 			}
 		case constants.KEEPTTL, constants.Keepttl:
 			keepttl = true
+		case "GET":
+			// Get the key from the hash table
+			obj := Get(key)
```

Comment:
An error is returned and SET aborted if the value stored at key is not a string.

refer: https://redis.io/docs/latest/commands/set/

Reply:
@JyotinderSingh , I've a doubt, that by default dice stores everything as a string only, right? Then why this error is being shown?

Reply:
The behavior should be correct from the implementation perspective. So we should do the check on our side.

Reply:
Oh! Okay 👍 
BTW @JyotinderSingh , I wanted to ask that could you please provide a sample test case like when the test is failing? like I've a doubt that how can I set the value to int or something else (anything other than string)?

If possible, may I give it a try please, which is causing this problem! (like if the value is not string and then it's creating a problem). Like, if anyone else is already working on it, then I can collaborate with them 🙏 ?